Sources/Sentry/SentryProfiler.mm

```diff
@@ -161,13 +169,41 @@
     }
 }
 
+NSArray *
+processFrameRenderInfo(SentryFrameInfoTimeSeries *frameInfo, uint64_t start, uint64_t duration)
+{
+    auto relativeFrameTimestampsNs = [NSMutableArray array];
+    [frameInfo enumerateObjectsUsingBlock:^(
+        NSDictionary<NSString *, NSNumber *> *_Nonnull obj, NSUInteger idx, BOOL *_Nonnull stop) {
+        const auto begin = (uint64_t)(obj[@"start_timestamp"].doubleValue * 1e9);
```

Comment:
`l`: If SentryFrameInfoTimeSeries would be a class then we wouldn't need to cast around with dictionaries. If you agree, please do this in another PR.

Reply:
I agree with this, but I would start with trying a struct instead of an objc class 👍🏻